content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,240 @@
+========================================
+Connect Gmail to Odoo using Google OAuth
+========================================
+
+Odoo is compatible with Google's OAuth for Gmail. In order to send secure emails from a custom
+domain, configure a few settings on Google's *Workspace* platform, as well as on the back end of
+the Odoo database. This is necessary to use a Google external email server with both a personal
+(youname@gmail.com) and custom domain (name@yourcompany.com / Google Workspace Account).
```

Comment:
Line 8: you -> your; change `(youname@gmail.com)` to `(yourname@gmail.com)`

Reply:
Also, the fake emails in Line 8, when this RST file is rendered to the HTML page, are highlighted and have external link buttons so you can click on them to send an email to the fake emails, not sure if that can be fixed, but it would be nice if they were just plain text.

Reply:
@mivu-odoo Great point! I was able to prevent formatting the emails by escaping the @ character.  
For example: `example\@gmail.com`

[More info on escaping text](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#escaping-mechanism)